group_vars/tag_Env_Yugabyte_nodes_master.yml

```diff
@@ -0,0 +1,9 @@
+link_destinations:
+  - "{{ master_dir }}"
+  - "{{ tserver_dir }}"
+
+templates_names:
+  - "master"
+  - "tserver"
+
+
```

Comment:
just curious why do we have tserver in the master node tag?

Reply:
As per the [documentation guidelines] (https://docs.yugabyte.com/latest/deploy/public-clouds/aws/manual-deployment/#verify-2), one has to create symlinks for both tserver and tmaster servers to be able to update  in an easier manner pointing the symlink to a new version in case it is needed. Other reason I have seen as per the guidelines is to segregate the data that tservers produce and the data tmaster servers yield. Although, from my point of view, this could be a good starting point to only create in the future one symlink instead of two for both kind of servers.

As for master servers, the manual documentation guidelines indicate to initialize both: master as well as tservers nodes. On tserver nodes, it is only required to initialize tserver process.